<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shoppinglistClass</class>
 <widget class="QWidget" name="shoppinglistClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>shoppinglist</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="itemsListWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Search:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchLineEdit">
         <property name="placeholderText">
          <string>Enter substring...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListWidget" name="sortedItemsListWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="categoryLineEdit">
         <property name="placeholderText">
          <string>Enter category...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showButton">
         <property name="text">
          <string>Show items</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="shoppinglist.qrc"/>
 </resources>
 <connections/>
</ui>
